shared/common-adapters/clickable-box.native.js

```diff
@@ -22,7 +22,7 @@ class ClickableBox extends React.Component<Props> {
             pointerEvents={props.pointerEvents}
             style={clickStyle}
             underlayColor={props.underlayColor || globalColors.white}
-            activeOpacity={0.7}
+            activeOpacity={this.props.activeOpacity ?? 0.7}
```

Comment:
This prop always existed but was unused. There are a couple places where we set it to .6 or .8, I don't think this will affect things much but I can remove them if someone wants, cc @adamjspooner?

Reply:
This is just the opacity of the container when it's being pressed, right?

Reply:
Yep

Reply:
LGTM!